sktime/forecasting/compose/_fallback.py

```diff
@@ -0,0 +1,254 @@
+            If all forecasters fail to fit.
+        """
+        self.first_nonfailing_forecaster_index_ = 0
+        self.exceptions_raised_ = dict()
+        return self._try_fit_forecasters(y=y, X=X, fh=fh)
+
+    def _try_fit_forecasters(self, y, X, fh):
+        """
+        Attempt to fit the forecasters in sequence until one succeeds.
+
+        This method iterates over the forecasters starting from the index
+        `first_nonfailing_forecaster_index_`. For each forecaster, it tries to fit it
+        with the current data. If the fit method of a forecaster raises an exception,
+        it records the exception and proceeds to the next forecaster. If a forecaster
+        fits successfully, it updates the current forecaster and its name.
+
+        Returns
+        -------
+        self : an instance of self
+
+        Raises
+        ------
+        RuntimeError
+            If all forecasters fail to fit.
+        """
+        while True:
+            ix = self.first_nonfailing_forecaster_index_
+            if self.first_nonfailing_forecaster_index_ >= len(self.forecasters_):
+                raise RuntimeError("No remaining forecasters to attempt prediction.")
+            name, forecaster = self.forecasters_[ix]
+            try:
+                self.current_name_ = name
+                self.current_forecaster_ = forecaster.clone()
+                self.current_forecaster_.fit(y=y, X=X, fh=fh)
+                return self
+            except Exception as e:
+                self.exceptions_raised_[self.first_nonfailing_forecaster_index_] = {
+                    "failed_at_step": "fit",
+                    "exception": e,
+                    "forecaster_name": name,
+                }
+                self.first_nonfailing_forecaster_index_ += 1
+                if self.verbose:
+                    warn(
+                        f"Forecaster {name} failed to fit with error: {e}",
+                        stacklevel=2,
+                        obj=self,
+                    )
+
+    def _predict(self, fh, X=None):
+        """Predict using the current forecaster.
+
+        If predict fails, fit and predict with the next forecaster.
+
+        Parameters
+        ----------
+        fh : array-like
+            The forecasting horizon.
+        X : array-like, optional (default=None)
+            Exogenous variables.
+
+        Returns
+        -------
+        y_pred : array-like
+            The predicted values.
+
+        Raises
+        ------
+        RuntimeError
+            If no forecaster is fitted or all forecasters fail to predict.
+        """
+        if self.current_forecaster_ is None:
+            raise RuntimeError("No forecaster has been successfully fitted yet.")
+
+        try:
+            return self.current_forecaster_.predict(fh, X)
+        except Exception as e:
+            self.exceptions_raised_[self.first_nonfailing_forecaster_index_] = {
+                "failed_at_step": "fit",
```

Comment:
Shouldn't it be "predict"?

Reply:
@yarnabrina Good catch, it should be `predict`